pkg/docker/util.js

```diff
@@ -595,6 +595,63 @@
         return deferred.promise();
     };
 
+    util.delete_image_confirm = function confirm(client, image) {
+        var deferred = $.Deferred();
+        var image_id = image.image_id !== undefined ? image.image_id : image.Id;
+        var image_name = image.name !== undefined ? image.name :
+            util.render_container_name(image.RepoTags[0]);
+
+        $('#delete-image-confirmation-dialog-title').text(
```

Comment:
`document.getElementById("delete-image-confirmation-dialog-title").textContent = ...` is a standard JS equivalent without jQuery.